datumaro/plugins/imagenet_txt_format.py

```diff
@@ -87,17 +102,34 @@ def build_cmdline_parser(cls, **kwargs):
         parser = super().build_cmdline_parser(**kwargs)
         parser.add_argument('--labels-file', dest='labels',
             help="Path to the file with label descriptions (synsets.txt)")
+        parser.add_argument('--no-labels-file', dest='labels',
+            action='store_const', const=(),
+            help="Do not load a file with label descriptions")
```

Comment:
Consider changing the option to be `--labels` and accept value `auto` or making a mutually-exclusive group. Otherwise, the behavior can be undefined or obscure, at least.